src/app/functions/server/onerep.ts

```diff
@@ -376,7 +342,7 @@ export async function isEligibleForFreeScan(
   const scanResult = await getLatestOnerepScanResults(profileId);
 
   if (scanResult.scan) {
-    logger.warn("User has already used free scan");
+    logger.info("User has already used free scan");
```

Comment:
Q: Do we have a convention on when we log slugs vs strings?

Re: 
```js
logger.info("deactivated_onerep_profile", {...});
logger.info("User has already used free scan");
```

Not sure if it really matters.
PROS: Consistency
CONS: ? Might break existing metrics for a while?

Reply:
@mansaj and I have discussed the need for it - I think we should go underscore style in general since it makes it better for metrics (it can be treated as an event name with parameters, vs. just a string)

Reply:
Mostly for my own future reference&hellip; but also, sorry for the _far_ too deep chain of <kbd>grep</kbd>  commands. It sickens me and brings me great shame. 😞 

```
git grep -Pino "\blogger\.\w+\([\"'][\w\s_-]+[\"']" src/ | grep -Eo "['\"].*['\"]" | grep -v "_" | grep -Ev "^[a-z\"]+$" | sort | uniq -c

   2 "No user ID for Nimbus telemetry"
   1 "No user ID for telemetry"
   1 "Successfully queued breach notification"
   1 "User has already used free scan"
   2 "addAttributionForSubscriber"
   1 "addFeatureFlag"
   1 "deleteAttributionById"
   1 "deleteAttributionsForSubscriber"
   1 "deleteFeatureFlagByName"
   1 "disableFeatureFlagByName"
   1 "enableFeatureFlagByName"
   1 "fxa-confirmed-profile-data"
   3 "fxaRpEvents"
   1 "getAttributionsForSubscriber"
   1 "getFeatureFlagByName"
   1 "getJwtPubKey"
   1 "getLatestAttributionForSubscriberWithType"
   1 "updateAllowList"
   1 "updateAttribution"
   1 "updateDependencies"
   1 "updateOwner"
   1 "updateWaitList"
```